docs/day1/Go-To-Build.md

```diff
@@ -1,88 +0,0 @@
-
-## Background
-
-When a new release goes to build, we need to track its progress in releasewarrior to ensure the right things happen at the right time. Following these instructions adds a release to releasewarrior.
-
-## Prerequisites
-- [releasewarrior-2.0](https://github.com/mozilla-releng/releasewarrior-2.0#installing) installed and working, with write access to that repository.
-
-
-## When to perform these steps
-
-The release-signoff mailing list will receive a message alerting us to a new build being created in ship-it. Here is an example email:
-
-    
-    subject: [desktop] Build of Firefox-58.0b5-build1
-
-    A new build has been submitted through ship-it:
-
-    Commit: https://hg.mozilla.org/releases/mozilla-beta/rev/f155e109bb419696beb422b5afbfd7299b2b2500
-    Task group: https://tools.taskcluster.net/push-inspector/#/C8umMF6qSKy3X1tntYk1pA
-    Locales: https://ship-it.mozilla.org/releases/Firefox-58.0b5-build1/l10n (requires VPN access)
-
-    Created by release manager @ mozilla.com
-    Started by release manager @ mozilla.com
-
-    Comparing Mercurial tag FIREFOX_58_0b4_RELEASE to FIREFOX_58_0b5_RELEASE:
-    Bugs since previous changeset: https://mzl.la/2j8Tbfe
-    Full Mercurial changelog: https://hg.mozilla.org/releases/mozilla-beta/pushloghtml?
-    fromchange=FIREFOX_58_0b4_RELEASE&tochange=f155e109bb419696beb422b5afbfd7299b2b2500&full=1
-
-## What to do
```

Comment:
I think the information in this file about setting things up in releasewarrior is still useful. This file could probably be cleaned up, as the above mail isn't sent for beta's anymore (see [Bug 1492128](https://bugzilla.mozilla.org/show_bug.cgi?id=1492128)), but I suggest landing this change without touching this file, and cleaning it up in a follow up.

Reply:
Sounds good, I restored the file.